bin/cli.js

```diff
@@ -28,24 +28,24 @@ const undeployGoal = require('../lib/goals/undeploy');
 module.exports = async function run (options) {
   try {
     const config = await nodeshiftConfig(options);
-    let enrichedResources = {};
+    const response = {};
     if (options.cmd === 'resource' || options.cmd === 'deploy' || options.cmd === 'apply-resource') {
-      enrichedResources = await resourceGoal(config);
+      const enrichedResources = await resourceGoal(config);
+      response.resources = enrichedResources;
+      if (options.cmd === 'deploy' || options.cmd === 'apply-resource') {
+        response.appliedResources = await applyResources(config, enrichedResources);
+      }
     }
 
     if (options.cmd === 'deploy' || options.cmd === 'build') {
-      await buildGoal(config);
-    }
-
-    if (options.cmd === 'deploy' || options.cmd === 'apply-resource') {
-      await applyResources(config, enrichedResources);
+      response.build = await buildGoal(config);
```

Comment:
speaking of Imagestreams, should we do anything with outputting the imagestream that we create.

if i remember correctly, the f-m-p outputs the created imagestream's template to a file, but i was never really certain why.  i suppose we could always add it later if we needed it

Reply:
Hmm - I'm not sure what the benefit would be. Here is the `ImageStream` YAML from my working environment for `nodejs-rest-http`. Not sure what is here that I would necessarily need. (Note there are lots and lots of images, because I was rebuilding over and over again yesterday)

<details><summary>nodejs-rest-http-image-stream.yaml</summary>
<pre><code>
apiVersion: v1
kind: ImageStream
metadata:
  creationTimestamp: '2017-11-29T19:52:48Z'
  generation: 1
  labels:
    project: nodejs-rest-http
    provider: nodeshift
    version: 1.0.3
  name: nodejs-rest-http
  namespace: testing-nodeshift
  resourceVersion: '248337'
  selfLink: /oapi/v1/namespaces/testing-nodeshift/imagestreams/nodejs-rest-http
  uid: e00207a3-d53e-11e7-aaa1-4613abbd5582
spec:
  lookupPolicy:
    local: false
status:
  dockerImageRepository: '172.30.1.1:5000/testing-nodeshift/nodejs-rest-http'
  tags:
    - items:
        - created: '2017-12-05T22:47:46Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:22dc897a7bf7a680b893a1948e293b74b9805697db56b21b9e1881c77d57acd5
          generation: 1
          image: >-
            sha256:22dc897a7bf7a680b893a1948e293b74b9805697db56b21b9e1881c77d57acd5
        - created: '2017-12-05T22:45:45Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:819c7a86a31b951410d43a95e108150ff3b3a4fc42d65d0d8753c5b302dcaa77
          generation: 1
          image: >-
            sha256:819c7a86a31b951410d43a95e108150ff3b3a4fc42d65d0d8753c5b302dcaa77
        - created: '2017-12-05T22:43:22Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:00fc48465717841e5afdadd729f70384e944fedc424c3f6f16d70b529a0ad378
          generation: 1
          image: >-
            sha256:00fc48465717841e5afdadd729f70384e944fedc424c3f6f16d70b529a0ad378
        - created: '2017-12-05T22:42:50Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:6d8bff2092d46dd87f96d3d6050a49845fd9cc502086c0fcdfddd6ad4d29186f
          generation: 1
          image: >-
            sha256:6d8bff2092d46dd87f96d3d6050a49845fd9cc502086c0fcdfddd6ad4d29186f
        - created: '2017-12-05T22:41:57Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:376657fbf5a7435c2c6f8c55cb638035d7c0e7209cc2e4d344fb39395d59e5d3
          generation: 1
          image: >-
            sha256:376657fbf5a7435c2c6f8c55cb638035d7c0e7209cc2e4d344fb39395d59e5d3
        - created: '2017-12-05T22:35:05Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:c97b21a010e7198870cf127c0a7da290dca16700484e1651c5ab8322a7dfebb9
          generation: 1
          image: >-
            sha256:c97b21a010e7198870cf127c0a7da290dca16700484e1651c5ab8322a7dfebb9
        - created: '2017-12-05T22:34:28Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:a3802528b6d3907b8adde1a4a6d638f275966faaa77f8ede0f66370e5927acbe
          generation: 1
          image: >-
            sha256:a3802528b6d3907b8adde1a4a6d638f275966faaa77f8ede0f66370e5927acbe
        - created: '2017-12-05T22:33:39Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:15a73deb0a590c961b6f122ff524ba91b23bcd692a1063de88ed43c7b270bdab
          generation: 1
          image: >-
            sha256:15a73deb0a590c961b6f122ff524ba91b23bcd692a1063de88ed43c7b270bdab
        - created: '2017-12-05T22:33:07Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:7f43020057b7ed576f8fdf1f62c9a4aac53d9a9cd737c2d7f493d9b65b9f4f07
          generation: 1
          image: >-
            sha256:7f43020057b7ed576f8fdf1f62c9a4aac53d9a9cd737c2d7f493d9b65b9f4f07
        - created: '2017-12-05T22:32:13Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:bf3ff341e117e8858223a7935a2c64d89708c7343c6dce26df4e64a45563f801
          generation: 1
          image: >-
            sha256:bf3ff341e117e8858223a7935a2c64d89708c7343c6dce26df4e64a45563f801
        - created: '2017-12-05T22:31:15Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:4a3ca732dc42c69d1bdb08fa7d60fc531289c5f5f1bd436a2020279afd3f59fa
          generation: 1
          image: >-
            sha256:4a3ca732dc42c69d1bdb08fa7d60fc531289c5f5f1bd436a2020279afd3f59fa
        - created: '2017-12-05T22:30:11Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:2102274ae6f56f3cb143426cb815d80f45f6fc28bc0ed0fd228385af84e686d6
          generation: 1
          image: >-
            sha256:2102274ae6f56f3cb143426cb815d80f45f6fc28bc0ed0fd228385af84e686d6
        - created: '2017-12-05T22:28:58Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:a6759cc2b219877fc8ea6761bfb729f455b253cbcfbe5bbac484004797e81401
          generation: 1
          image: >-
            sha256:a6759cc2b219877fc8ea6761bfb729f455b253cbcfbe5bbac484004797e81401
        - created: '2017-12-05T22:28:11Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:6cfcc5d28d5d25726001fc0ecfb771d2830c6a8dab61c54405230eb27c4cdbde
          generation: 1
          image: >-
            sha256:6cfcc5d28d5d25726001fc0ecfb771d2830c6a8dab61c54405230eb27c4cdbde
        - created: '2017-12-05T22:27:47Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:9543cd2099814dc29cc33aa5e100997bb97655fdc7a73058cb5a5216dfc3b747
          generation: 1
          image: >-
            sha256:9543cd2099814dc29cc33aa5e100997bb97655fdc7a73058cb5a5216dfc3b747
        - created: '2017-12-05T22:27:06Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:8ea5000212a5b76ca887e0fa1805de99a5b34a2207afa2c31971054fd4df1e9d
          generation: 1
          image: >-
            sha256:8ea5000212a5b76ca887e0fa1805de99a5b34a2207afa2c31971054fd4df1e9d
        - created: '2017-12-05T22:26:42Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:5369f4ef72d997987f5e0d068649b817cb929897a3f1dea2257575917a4ce1e1
          generation: 1
          image: >-
            sha256:5369f4ef72d997987f5e0d068649b817cb929897a3f1dea2257575917a4ce1e1
        - created: '2017-12-05T22:25:36Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:ad4476cd76298405957264c02fdb4bbd6f2879830d59dbd03906bd98fd709a51
          generation: 1
          image: >-
            sha256:ad4476cd76298405957264c02fdb4bbd6f2879830d59dbd03906bd98fd709a51
        - created: '2017-12-05T22:24:28Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:600624727dbb0a20e6f08ac4c243a1abdc50582015ed68251710977440cc0f10
          generation: 1
          image: >-
            sha256:600624727dbb0a20e6f08ac4c243a1abdc50582015ed68251710977440cc0f10
        - created: '2017-12-05T22:21:02Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:a70a9cba3a685f1662ba95d3918e9b54594b010e96de8facd84edcffa8442ef2
          generation: 1
          image: >-
            sha256:a70a9cba3a685f1662ba95d3918e9b54594b010e96de8facd84edcffa8442ef2
        - created: '2017-12-05T21:41:11Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:3e2d51825469a58e092cc2c165739ae2d450416fbe8454bc791c1857960251e7
          generation: 1
          image: >-
            sha256:3e2d51825469a58e092cc2c165739ae2d450416fbe8454bc791c1857960251e7
        - created: '2017-12-05T21:26:11Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:8fd54ad51f0958476760e83cc10891df8dec2722eb80ae793a0166523ab6ceb5
          generation: 1
          image: >-
            sha256:8fd54ad51f0958476760e83cc10891df8dec2722eb80ae793a0166523ab6ceb5
        - created: '2017-12-05T21:25:01Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:1e4577d0ef660231f36e40793b309b58ae5955816b1dd1316b93909ce0362b89
          generation: 1
          image: >-
            sha256:1e4577d0ef660231f36e40793b309b58ae5955816b1dd1316b93909ce0362b89
        - created: '2017-12-05T21:24:33Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:2068905a9ceb980b8edff46d126bbcf4aab6c1a9cb9c37195f1d986af7823c65
          generation: 1
          image: >-
            sha256:2068905a9ceb980b8edff46d126bbcf4aab6c1a9cb9c37195f1d986af7823c65
        - created: '2017-12-05T21:21:21Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:83f3d243f0fccac42a1575fd6d06795bb21c77b8168e6aac8eef78c9b5191493
          generation: 1
          image: >-
            sha256:83f3d243f0fccac42a1575fd6d06795bb21c77b8168e6aac8eef78c9b5191493
        - created: '2017-12-05T21:20:41Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:bbdc68b0242a9a7b3b3ebd738fa930cf2cded88c8c1ab0da11f375d55b0c90a4
          generation: 1
          image: >-
            sha256:bbdc68b0242a9a7b3b3ebd738fa930cf2cded88c8c1ab0da11f375d55b0c90a4
        - created: '2017-12-05T21:20:13Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:447fe70a70bfde2ca7638956781bed4eef3d9dd9311ef97ca5455a5889320988
          generation: 1
          image: >-
            sha256:447fe70a70bfde2ca7638956781bed4eef3d9dd9311ef97ca5455a5889320988
        - created: '2017-12-05T21:19:25Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:e6ac17a6833fab5ffc2b55bc0f2c966cd736fd335783e62756d62b26598e1fe6
          generation: 1
          image: >-
            sha256:e6ac17a6833fab5ffc2b55bc0f2c966cd736fd335783e62756d62b26598e1fe6
        - created: '2017-12-05T21:18:41Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:38654ea5127b1051fe384ed05fc3aad7485c6934bf8dc8db6881003c1a6718c6
          generation: 1
          image: >-
            sha256:38654ea5127b1051fe384ed05fc3aad7485c6934bf8dc8db6881003c1a6718c6
        - created: '2017-12-05T21:17:38Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:0875250b83046aca315e6f91402a78c687acaebb7c8c2084cc1d7fd35f98b53c
          generation: 1
          image: >-
            sha256:0875250b83046aca315e6f91402a78c687acaebb7c8c2084cc1d7fd35f98b53c
        - created: '2017-12-05T21:16:30Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:e05c0ce027372cc5bf1e68f9c2d599e7337bf6312088e953323c765b2a63cde8
          generation: 1
          image: >-
            sha256:e05c0ce027372cc5bf1e68f9c2d599e7337bf6312088e953323c765b2a63cde8
        - created: '2017-12-05T21:15:22Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:a4672d69a10ca14722a727edcaa41c574e8e98d7d639c0e358ab6d8b58dc6476
          generation: 1
          image: >-
            sha256:a4672d69a10ca14722a727edcaa41c574e8e98d7d639c0e358ab6d8b58dc6476
        - created: '2017-12-05T21:13:54Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:bafb1086865f3b683c45f9bb3c5d912e45be46ccbff907b3d23bfb3819a1c2ef
          generation: 1
          image: >-
            sha256:bafb1086865f3b683c45f9bb3c5d912e45be46ccbff907b3d23bfb3819a1c2ef
        - created: '2017-12-05T21:10:48Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:31cb6bcf9b8e713ec500d1a3032faecdb17ec8fb5f56c1afaa249602839306aa
          generation: 1
          image: >-
            sha256:31cb6bcf9b8e713ec500d1a3032faecdb17ec8fb5f56c1afaa249602839306aa
        - created: '2017-12-05T21:02:37Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:7bf2ca71cf54dfef664a404d05dbef71c65d277e15389c699968be277962f537
          generation: 1
          image: >-
            sha256:7bf2ca71cf54dfef664a404d05dbef71c65d277e15389c699968be277962f537
        - created: '2017-12-05T20:00:48Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:c3fd6c8095be3c52d2ae0e1d4ef89d635ddca1194223a342896f0374bec73656
          generation: 1
          image: >-
            sha256:c3fd6c8095be3c52d2ae0e1d4ef89d635ddca1194223a342896f0374bec73656
        - created: '2017-12-05T19:58:55Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:f3be8c9e841f639ba144dbb7a5b8de91c7f11c271b6194e895f7140a4fc8ea30
          generation: 1
          image: >-
            sha256:f3be8c9e841f639ba144dbb7a5b8de91c7f11c271b6194e895f7140a4fc8ea30
        - created: '2017-12-05T19:53:20Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:dea8fa57c56b35a42bdf69f6f88575de368def85b117c1df8e8cbf92f4ba12a8
          generation: 1
          image: >-
            sha256:dea8fa57c56b35a42bdf69f6f88575de368def85b117c1df8e8cbf92f4ba12a8
        - created: '2017-12-05T19:50:44Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:94a3e914a192b23fecbcb6b9ff20b8173d5aa6d3aed4514de5c43e4ef99579c6
          generation: 1
          image: >-
            sha256:94a3e914a192b23fecbcb6b9ff20b8173d5aa6d3aed4514de5c43e4ef99579c6
        - created: '2017-11-30T21:48:59Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:50917cf57038303f725bba7981fa2d412bf3bf1e2f723ccb0e006451b28732bd
          generation: 1
          image: >-
            sha256:50917cf57038303f725bba7981fa2d412bf3bf1e2f723ccb0e006451b28732bd
        - created: '2017-11-30T21:48:06Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:3650919ef29dff6d4ad61ea951231874a483b8216925c5e8a11b7de96c47d0e0
          generation: 1
          image: >-
            sha256:3650919ef29dff6d4ad61ea951231874a483b8216925c5e8a11b7de96c47d0e0
        - created: '2017-11-30T21:47:01Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:aedfa2d58c24952b5c61923b7a1467dc919e7d6d611cca49a4b76792c1da8bd1
          generation: 1
          image: >-
            sha256:aedfa2d58c24952b5c61923b7a1467dc919e7d6d611cca49a4b76792c1da8bd1
        - created: '2017-11-30T21:46:03Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:d68df12c84462748822e54922ffb2e0b70665e1cd0d3f98437b7557e3a3cbbc9
          generation: 1
          image: >-
            sha256:d68df12c84462748822e54922ffb2e0b70665e1cd0d3f98437b7557e3a3cbbc9
        - created: '2017-11-30T21:43:07Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:ba5ddae29430c884b3fec7fca1c8174a6edebcb91aa017419d8aa478dbf3e5c0
          generation: 1
          image: >-
            sha256:ba5ddae29430c884b3fec7fca1c8174a6edebcb91aa017419d8aa478dbf3e5c0
        - created: '2017-11-30T21:39:24Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:7c18baf4e36de0541e721a5e111e3a36f980ca13c42d5389789362b3151a3f2e
          generation: 1
          image: >-
            sha256:7c18baf4e36de0541e721a5e111e3a36f980ca13c42d5389789362b3151a3f2e
        - created: '2017-11-30T21:38:10Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:24110c83b66eba2ad2ff2b31a413c94dbefa8aeac91c0829b3dc450f60b3d966
          generation: 1
          image: >-
            sha256:24110c83b66eba2ad2ff2b31a413c94dbefa8aeac91c0829b3dc450f60b3d966
        - created: '2017-11-30T21:33:53Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:be4c1935c4745d18a317bb347e9c3decaca744058f8b726b4cecad589bc0c27d
          generation: 1
          image: >-
            sha256:be4c1935c4745d18a317bb347e9c3decaca744058f8b726b4cecad589bc0c27d
        - created: '2017-11-30T21:07:52Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:6047566289b9dc7b6102d81e9c7d593ec104bf3ce674152adb399e61f5275f4c
          generation: 1
          image: >-
            sha256:6047566289b9dc7b6102d81e9c7d593ec104bf3ce674152adb399e61f5275f4c
        - created: '2017-11-29T21:00:12Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:c04f7766bf6c95649d6ae4a55f9fae0f4b619d1ff4cda306dd46b2d655c0f800
          generation: 1
          image: >-
            sha256:c04f7766bf6c95649d6ae4a55f9fae0f4b619d1ff4cda306dd46b2d655c0f800
        - created: '2017-11-29T20:57:23Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:268f946384ed4456ea742b55f40a6b6d1936e595683a658f52b22b6997a3428d
          generation: 1
          image: >-
            sha256:268f946384ed4456ea742b55f40a6b6d1936e595683a658f52b22b6997a3428d
        - created: '2017-11-29T20:56:37Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:5b09fe18824e12c8a6f576a0beae4cf146871ed7a8e0d71c1b406afad05d1485
          generation: 1
          image: >-
            sha256:5b09fe18824e12c8a6f576a0beae4cf146871ed7a8e0d71c1b406afad05d1485
        - created: '2017-11-29T20:55:58Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:b016c8d5cc3bf9f6161b9788cc5ec7ba64bc330faee9104c597f753a68b99d95
          generation: 1
          image: >-
            sha256:b016c8d5cc3bf9f6161b9788cc5ec7ba64bc330faee9104c597f753a68b99d95
        - created: '2017-11-29T20:53:12Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:c30c81d9ce872b9c0815f7ef5721d153c8192ed931d5ad3838e2b66cffbab0ce
          generation: 1
          image: >-
            sha256:c30c81d9ce872b9c0815f7ef5721d153c8192ed931d5ad3838e2b66cffbab0ce
        - created: '2017-11-29T20:48:47Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:73a4e219116003f850903773110938ba84c374567185f671dafebf2463baa3b4
          generation: 1
          image: >-
            sha256:73a4e219116003f850903773110938ba84c374567185f671dafebf2463baa3b4
        - created: '2017-11-29T20:48:09Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:ca39d94d91e5082b5b1bdb6b83a4320cd7bd04894d5d4c5ba867cfc1b3ac0140
          generation: 1
          image: >-
            sha256:ca39d94d91e5082b5b1bdb6b83a4320cd7bd04894d5d4c5ba867cfc1b3ac0140
        - created: '2017-11-29T20:39:16Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:1caf04398a077900d80ab693b186613e0566b3ad20cf178c236d65e2b76295e3
          generation: 1
          image: >-
            sha256:1caf04398a077900d80ab693b186613e0566b3ad20cf178c236d65e2b76295e3
        - created: '2017-11-29T20:28:48Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:c7fca3df6fef2a80f913efe5f60ab90aa9d9b961499712c96d8ca58d7fcefc1d
          generation: 1
          image: >-
            sha256:c7fca3df6fef2a80f913efe5f60ab90aa9d9b961499712c96d8ca58d7fcefc1d
        - created: '2017-11-29T20:24:36Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:fd610893513f8c737434ca4365056e8b8619d3635c4b48f994011ea6eb9cc452
          generation: 1
          image: >-
            sha256:fd610893513f8c737434ca4365056e8b8619d3635c4b48f994011ea6eb9cc452
        - created: '2017-11-29T20:23:56Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:addbe31c8ab6ce61b7ede0696dfd61c8ecc4f0101acb02ce24d83baa01c2b288
          generation: 1
          image: >-
            sha256:addbe31c8ab6ce61b7ede0696dfd61c8ecc4f0101acb02ce24d83baa01c2b288
        - created: '2017-11-29T20:22:41Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:af1237aa8b713d57e0aaf69aa285323a9dc8eda745a15cc067e5c38e82a38d77
          generation: 1
          image: >-
            sha256:af1237aa8b713d57e0aaf69aa285323a9dc8eda745a15cc067e5c38e82a38d77
        - created: '2017-11-29T20:18:02Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:3c40bbee6ec61360418a19817800cbf2108c618fd528777a5928e22649d7d4d0
          generation: 1
          image: >-
            sha256:3c40bbee6ec61360418a19817800cbf2108c618fd528777a5928e22649d7d4d0
        - created: '2017-11-29T20:09:16Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:ff91ff3497ab162f2eaced4e4e596d3a0e4a924953ccf110c930c95c8d31d530
          generation: 1
          image: >-
            sha256:ff91ff3497ab162f2eaced4e4e596d3a0e4a924953ccf110c930c95c8d31d530
        - created: '2017-11-29T20:08:20Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:22b7e331800f2caedd8f6720c0dafd43ff298a25b3a795ad00f696b7179c1fff
          generation: 1
          image: >-
            sha256:22b7e331800f2caedd8f6720c0dafd43ff298a25b3a795ad00f696b7179c1fff
        - created: '2017-11-29T20:07:04Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:6e43859aaba72c52a2cca8af89ad659db605bd643d7ea7f069b5f9ea9ce182af
          generation: 1
          image: >-
            sha256:6e43859aaba72c52a2cca8af89ad659db605bd643d7ea7f069b5f9ea9ce182af
        - created: '2017-11-29T20:03:33Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:41176deef4e291339ae41e20c387cd1c192a3fca7303106c231f6650cb15b0f9
          generation: 1
          image: >-
            sha256:41176deef4e291339ae41e20c387cd1c192a3fca7303106c231f6650cb15b0f9
        - created: '2017-11-29T20:02:25Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:0b2f047d59efcc272e70caf5dacabaf0f7c67b4240450d5af074426cc883e559
          generation: 1
          image: >-
            sha256:0b2f047d59efcc272e70caf5dacabaf0f7c67b4240450d5af074426cc883e559
        - created: '2017-11-29T20:00:08Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:b580ad5c20cc92d6f3c05599851b40e3b2a2727cc138d47c0fc7abf1e64102a9
          generation: 1
          image: >-
            sha256:b580ad5c20cc92d6f3c05599851b40e3b2a2727cc138d47c0fc7abf1e64102a9
        - created: '2017-11-29T19:59:35Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:218c88e166bffe4cece35c23beffd34bb269580e373a08339c9ba97abea1fa03
          generation: 1
          image: >-
            sha256:218c88e166bffe4cece35c23beffd34bb269580e373a08339c9ba97abea1fa03
        - created: '2017-11-29T19:58:13Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:f7da727f2f49e7592b0f46ae5b9fc9f99c58fb680b68f4a83d4d54d84b6f70b7
          generation: 1
          image: >-
            sha256:f7da727f2f49e7592b0f46ae5b9fc9f99c58fb680b68f4a83d4d54d84b6f70b7
        - created: '2017-11-29T19:53:25Z'
          dockerImageReference: >-
            172.30.1.1:5000/testing-nodeshift/nodejs-rest-http@sha256:953409419a39c780cb3678385555c71310f138c952ccd29dec5498e81cf1d770
          generation: 1
          image: >-
            sha256:953409419a39c780cb3678385555c71310f138c952ccd29dec5498e81cf1d770
      tag: latest
</pre>
</code>
</details>

Reply:
yea, i don't really know why we would need it, so lets not add anything around that